### PR TITLE
Claim style improve 10

### PR DESCRIPTION
**Pull request by fairlighteth**
- Text changes
- Input field style changes

Screen Shot 2022-01-17 at 19 12 59

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @biocom , test and input field changes LGTM. 
However, I can't manually fill in the field..
Just one proposal: maybe we could make placeholder a bit more transparent as we do on the Swap form? 
placeholder
example

File: src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx [hunk 1]
```diff
@@ -183,16 +183,21 @@ export default function InvestOption({ approveData, updateInvestAmount, claim }:
...
-                {formatSmart(balance)} {currencyAmount?.currency?.symbol}
+                {formatSmart(balance) || 0} {currencyAmount?.currency?.symbol}
               </i>
               {/* Button should use the max possible amount the user can invest, considering their balance + max investment allowed */}
-              <button onClick={onMaxClick}>Invest max. possible</button>
+              <button onClick={onMaxClick}>(invest max. possible)</button>
             </span>
             <label>
+              <input
+                // disabled
```
Comment (W3stside) [inline src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx:195]: should it not be disabled via the attribute?

**Review by W3stside — APPROVED** [review]
nice

File: src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx (hunk 1 above)
Comment (biocom) [reply src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx:195]: My impression was this `disabled` attribute isn't set dynamically yet. And needed it to be not disabled for styling purposes. Commenting out as we need to address this still.

Comment (W3stside) [reply src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx:195]: got it, thought it was always disabled for this state, misread that. what are the criteria for disabled? we need to make an issue for this right?

**Pull request merged**